stable/mysql/templates/svc.yaml

```diff
@@ -7,6 +7,11 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+{{- if .Values.metrics.enabled }}
+  annotations:
```

Comment:
Don't hard-code the annotations. Just make them configurable. You don't need them if you use the prometheus-operator.